.github/workflows/wheels.yml

```diff
@@ -125,7 +125,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install -U pip
-          pip install -U numpy setuptools
+          pip install -U numpy setuptools<72
```

Comment:
```suggestion
          pip install -U numpy "setuptools<72"
```

Reply:
I didn't quite follow why these had to be installed first when they are anyway specified in the `requirements.txt`. Something is still trying to download the more recent `setuptools`.

Reply:
pyproject.toml.  Probably the pre-install isn't necessary anymore, but older versions of pip were pretty flaky when it came to installing things that were used for the build.  So it was safer to install numpy and setuptools by hand first to be sure.